tensorflow_io/core/kernels/kafka_kernels_deprecated.cc

```diff
@@ -242,6 +259,16 @@ class KafkaDatasetOp : public DatasetOpKernel {
                   }
                   out_tensors->emplace_back(std::move(key_tensor));
                 }
+                if (dataset()->message_offset_) {
+                  Tensor offset_tensor(cpu_allocator(), DT_STRING, {});
+                  int64_t offset = message->offset();
+                  int32_t partition = message->partition();
+                  string offset_string = std::to_string(partition);
+                  offset_string += string(":");
+                  offset_string += std::to_string(offset);
```

Comment:
maybe this would suffice?
```
string offset_string = std::to_string(partition) + ":" + std::to_string(offset);
```

Reply:
yes, this is more better , I'm going to adjust the code